UnityProject/Assets/Scripts/Tool/AtmosphericAnalyser.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AtmosphericAnalyser : MonoBehaviour, ICheckedInteractable<HandActivate>
+{
+
+	public bool WillInteract(HandActivate interaction, NetworkSide side)
+	{
+		if (!DefaultWillInteract.Default(interaction, side))
+		{
+			return false;
+		}
+
+		return true;
+	}
+
+	public void ServerPerformInteraction(HandActivate interaction)
+	{
+		string ToShow = "";
+		var MetaDataLayer = MatrixManager.AtPoint(interaction.PerformerPlayerScript.registerTile.WorldPositionServer, true).MetaDataLayer;
+		if (MetaDataLayer != null) { 
+			var Node = MetaDataLayer.Get(interaction.Performer.transform.localPosition.RoundToInt());
+			if(Node != null){
+				ToShow = "Pressure : " + Node.GasMix.Pressure + " Kpa \n"
```

Comment:
When I use atmos scanner out in space I get "NaN%" showing up for some of these. I think it should say 0 instead.

Reply:
if you want to use it for debugging it should show NaN%?

Reply:
Well it's an in-game item which is gonna get used by players too

Reply:
should I just replace it with non-existent?

Reply:
is 0 now